metadata/service/aggregator/single.yml

```diff
@@ -6,3 +6,4 @@ parameters:
   heka:
     aggregator:
       enabled: true
+      timezone: ${_param:linux_timezone}
```

Comment:
Not sure about this. This will fail if `_param:linux_ timezone` is not defined in the user model, meaning this is making `_param:linux_timezone` a mandatory reclass param.